src/modules/poweraccent/PowerAccent.Core/Services/SettingsService.cs

```diff
@@ -0,0 +1,180 @@
+                    if (settings != null)
+                    {
+                        ActivationKey = settings.Properties.ActivationKey;
+                        switch (settings.Properties.ToolbarPosition.Value)
+                        {
+                            case "Top center":
+                                Position = Position.Top;
+                                break;
+                            case "Bottom center":
+                                Position = Position.Bottom;
+                                break;
+                            case "Left":
+                                Position = Position.Left;
+                                break;
+                            case "Right":
+                                Position = Position.Right;
+                                break;
+                            case "Top right corner":
+                                Position = Position.TopRight;
+                                break;
+                            case "Top left corner":
+                                Position = Position.TopLeft;
+                                break;
+                            case "Bottom right corner":
+                                Position = Position.BottomRight;
+                                break;
+                            case "Bottom left corner":
+                                Position = Position.BottomLeft;
+                                break;
+                            case "Center":
+                                Position = Position.Center;
+                                break;
+                        }
+                    }
+                }
+                catch (Exception ex)
+                {
+                    Logger.LogError("Failed to read changed settings", ex);
+                }
+            }
+        }
+    }
+
+    private PowerAccentActivationKey _activationKey = PowerAccentActivationKey.LeftRightArrow;
+
+    public PowerAccentActivationKey ActivationKey
+    {
+        get
+        {
+            return _activationKey;
+        }
+
+        set
+        {
+            _activationKey = value;
+        }
+    }
+
+    private Position _position = Position.Top;
+
+    public Position Position
+    {
+        get
+        {
+            return _position;
+        }
+
+        set
+        {
+            _position = value;
+        }
+    }
+
+    private int _inputTime = 200;
+
+    public int InputTime
+    {
+        get
+        {
+            return _inputTime;
+        }
+
+        set
+        {
+            _inputTime = value;
+        }
+    }
+
+    public char[] GetLetterKey(LetterKey letter)
+    {
+        return GetDefaultLetterKey(letter);
+    }
+
+    public static char[] GetDefaultLetterKey(LetterKey letter)
+    {
+        switch (letter)
+        {
+            case LetterKey.A:
+                return new char[] { 'à', 'â', 'á', 'ä', 'ã', 'å' };
```

Comment:
```suggestion
                return new char[] { 'à', 'â', 'á', 'ä', 'ã', 'å', 'æ' };
```

Reply:
We have € on e. What about adding @ to a?

Reply:
I think we're going a bit too much past accents there.
I imagine every keyboard will have some way to do @.
Not even sure € should be there to be honest :(
Let's let community ask for adds later on.